src/multi_vector_simulator/D2_model_constraints.py

```diff
@@ -439,27 +445,22 @@ def constraint_minimal_degree_of_autonomy(model, dict_values, dict_model):
     - Test_Constraints.test_benchmark_minimal_degree_of_autonomy()
     """
 
-    # Keys for dicts renewable_assets and non_renewable_assets
-    oemof_solph_object_asset = "object"
-    weighting_factor_energy_carrier = "weighting_factor_energy_carrier"
-    oemof_solph_object_bus = "oemof_solph_object_bus"
-
     logging.debug(f"Data considered for the minimal degree of autonomy constraint:")
 
     demands = prepare_demand_assets(
         dict_values,
         dict_model,
-        oemof_solph_object_asset,
-        weighting_factor_energy_carrier,
-        oemof_solph_object_bus,
+        OEMOF_SOLPH_OBJECT_ASSET,
+        WEIGHTING_FACTOR_ENERGY_CARRIER,
+        OEMOF_SOLPH_OBJECT_BUS,
```

Comment:
```suggestion
```
No need to use the constant variables defined at the top as arguments anymore. Also change in `def prepare_demand_assets(...)` and its docstrings